docs/clients/go-client-api.md

```diff
@@ -8,6 +8,8 @@ category: user guide
 
 To apply to different scenarios, TiKV provides [two types of APIs](../overview.md#two-types-of-apis) for developers: the Raw Key-Value API and the Transactional Key-Value API. This document uses two examples to guide you through how to use the two APIs in TiKV. The usage examples are based on multiple nodes for testing. You can also quickly try the two types of APIs on a single machine.
 
+**Warning**: Don’t use these two APIs together in the same cluster, or they will corrupt each other’s data.
```

Comment:
@siddontang Yes. In most cases, the present tense is used in user documentation.

Reply:
How about `may` as it does not always corrupt data.

Reply:
Then I suggest the word "might". In documentation, we use "might" instead of "may", because "may" has multiple meanings. "might" is a translation-friendly word.